spacy/lang/en/tokenizer_exceptions.py

```diff
@@ -139,26 +139,27 @@
 
 # W-words, relative pronouns, prepositions etc.
 
-for word in [
-    "who",
-    "what",
-    "when",
-    "where",
-    "why",
-    "how",
-    "there",
-    "that",
-    "this",
-    "these",
-    "those",
+for word, morph in [
+    ("who", None),
+    ("what", None),
+    ("when", None),
+    ("where", None),
+    ("why", None),
+    ("how", None),
+    ("there", None),
+    ("that", "Number=Sing|Person=3"),
+    ("this", "Number=Sing|Person=3"),
+    ("these", "Number=Plur|Person=3"),
+    ("those", "Number=Plur|Person=3"),
 ]:
     for orth in [word, word.title()]:
-        _exc[orth + "'s"] = [
-            {ORTH: orth, NORM: word},
-            {ORTH: "'s", NORM: "'s"},
-        ]
+        if morph != "Number=Plur|Person=3":
+            _exc[orth + "'s"] = [
+                {ORTH: orth, NORM: word},
+                {ORTH: "'s", NORM: "'s"},
+            ]
 
-        _exc[orth + "s"] = [{ORTH: orth, NORM: word}, {ORTH: "s"}]
+            _exc[orth + "s"] = [{ORTH: orth, NORM: word}, {ORTH: "s"}]
```

Comment:
You could still have `thiss`, right? Which would likely be more like a misspelling/typo than two actual tokens?

Reply:
This PR only prevents `these s` or `this re` from being split into two tokens.

A lot of these exceptions (in particular `id`) have many contexts where they weren't intended as two tokens, but this PR is just trying to fix this/that/these/those where there's no analysis that makes sense as two tokens.